src/stream/src/executor/integration_tests.rs

```diff
@@ -74,7 +77,7 @@ async fn test_merger_sum_aggr() {
             1,
         )
         .unwrap();
-        let (tx, rx) = channel();
+        let (tx, rx) = channel(INITIAL_PERMITS, BATCHED_PERMITS);
```

Comment:
How about adding a `channel_for_test` function for this?

Reply:
Sure👌